### DISCUSSION TITLE
龙猫在做的那个《探索起源》整合包，进度怎么样了

### DISCUSSION THREAD

**Discussion by xxtg666** [Meloong-Git/PCL#1948] [讨论]
自从[去年10月说至少得等到今年下半年](https://www.bilibili.com/read/cv19133430#:~:text=%E5%BE%97%E7%AD%89%E5%88%B0%E6%98%8E%E5%B9%B4%EF%BC%8C-,%E6%8C%87%E4%B8%8D%E5%AE%9A%E8%BF%98%E5%BE%97%E7%AD%89%E5%88%B0%E6%98%8E%E5%B9%B4%E4%B8%8B%E5%8D%8A%E5%B9%B4,-%E2%80%A6%E2%80%A6)之后，就再没消息了，想问问现在开发进度怎么样了，以及什么时候下一次公测
~~[前几次测试](https://www.bilibili.com/read/cv12494698?spm_id_from=333.999.0.0#:~:text=%E8%99%BD%E7%84%B6%E4%B8%80%E7%9B%B4%E7%A7%98%E8%80%8C%E4%B8%8D%E5%AE%A3%EF%BC%8C%E4%BD%86%E5%9C%A8%E8%BF%87%E5%8E%BB%E7%9A%84%E5%87%A0%E4%B8%AA%E6%9C%88%E4%B8%AD%EF%BC%8C%E5%AE%9E%E9%99%85%E4%B8%8A%E3%80%8A%E6%8E%A2%E7%B4%A2%E8%B5%B7%E6%BA%90%E3%80%8B%E5%B7%B2%E7%BB%8F%E5%89%8D%E5%89%8D%E5%90%8E%E5%90%8E%E5%BC%80%E5%B1%95%E4%BA%86%205%20%E6%AC%A1%E5%86%85%E6%B5%8B%E3%80%82%E5%87%A0%E5%A4%A9%E5%89%8D%EF%BC%8C%E6%88%91%E4%BB%AC%E5%8F%AC%E9%9B%86%E4%BA%86%E5%B0%91%E9%87%8F%E7%8E%A9%E5%AE%B6%E8%BF%9B%E8%A1%8C%E7%AC%AC%E4%B8%80%E6%AC%A1%E5%85%AC%E6%B5%8B%EF%BC%8C%E7%9B%AE%E5%89%8D%E6%B5%8B%E8%AF%95%E5%B7%B2%E7%BB%8F%E5%9C%86%E6%BB%A1%E7%BB%93%E6%9D%9F%E5%95%A6%E3%80%82)我都没玩到，下次龙猫能给个资格吗（~~

**Comment by LTCatt** [comment]
目前主要精力都在做独立游戏，探索起源暂时搁置了 Orz
Mod 有 Bug 但我修不了血压太高了……
